apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -483,8 +489,47 @@ export const renderWarningOutdated = ({
   </Wrapper>
 );
 
+// Quick fix: the error TransportStatusError with status code LOCKED_DEVICE should be catched
+// inside all the device actions and mapped to an event of type "lockedDevice".
+// With this fix, we can catch all the error that were not catched upstream.
+export const renderLockedDeviceError = ({
+  t,
+  device,
+  onRetry,
+}: {
+  t: TFunction;
```

Comment:
Since you are already using the `Trans` component here, you could avoid passing the t function completely 🤔

Reply:
Yeah - actually i don't know what is the standard way to go now, i thought it would be to use `useTranslation` but `Trans` is already used in this file - maybe i can remove all my change on `renderError` (and this `renderLockedDeviceError` that passes `t` and just use the `Trans` component ?

Reply:
WDYT ?

Reply:
these are not components so you are right not using a hook indeed

Reply:
So at the end, i updated the LLD `renderError` to only use `t` and not the `Trans` component